index.html

```diff
@@ -2905,6 +2915,168 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls or a <a>holder</a>
+has been designated to use one or more particular confirmation methods and to
+bind the <a>claims</a> about the <a>subject</a> to one or more of these
+confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
+and enables a <a>verifier</a> to validate that the <a>holder</a> presenting the
+<a>verifiable credential</a> has proven control of one or more of these
+confirmation methods when the <a>claims</a> bound to the confirmation method
+are presented.
```

Comment:
```suggestion
confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
around how a <a>verifier</a> can validate that the <a>holder</a> presenting the
<a>verifiable credential</a> has control of one or more of the
confirmation methods bound to the <a>claims</a> in the presented <a>verifiable credential</a>.
```

Again not sure. But this sentence also feels too long to me.

Reply:
```suggestion
confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
around how a <a>verifier</a> can validate that the <a>holder</a> is bound to a set of <a>claims</a> presented in a
<a>verifiable credential</a>.
```

maybe